gshoppingflux/gshoppingflux.php

```diff
@@ -1634,7 +1634,7 @@ public function renderInfo()
 		));
 	}
 
-	public function customGetNestedCategories($shop_id, $root_category = null, $id_lang = false, $active = true, $groups = null, $use_shop_restriction = true, $sql_filter = '', $sql_sort = '', $sql_limit = '')
```

Comment:
Still glad to see people continuing to use it. Even if it requires some add/mod to the code, as requested by some users since a while now (unfortunately I can't spend the time needed to implement new things *sic*)...